requirements.txt

```diff
@@ -1,4 +1,4 @@
-boto3 ==1.35.81
+boto3 ==1.35.99
```

Comment:
This seems like an upgrade while the PR title mentioned a downgrade.

Reply:
The downgrade was done in the rock, I changed it here just to make sure that both charm and rock uses the same.